deploy/openshift/deployment-validation-operator-olm.yaml

```diff
@@ -13,6 +13,9 @@ objects:
     image: ${IMAGE}:${IMAGE_TAG}
     displayName: Deployment Validation Operator
     publisher: ${CATALOG_PUBLISHER}
+    updateStrategy:
+      registryPoll:
+        interval: 15m
```

Comment:
Let's templatize this and default to a value that will disable the polling if we can.  We would want someone to opt in so they know it is happening on their cluster.

Reply:
changed to `24h`

Reply:
This has been templatized